services/movie.py

```diff
@@ -0,0 +1,46 @@
+from db.models import Movie
+from typing import Optional
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: Optional[list[int]] = None,
+        actors_ids: Optional[list[int]] = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        movies = Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+
+    if genres_ids and not actors_ids:
+        movies = Movie.objects.filter(genres__id__in=genres_ids)
+    if actors_ids and not genres_ids:
+        movies = Movie.objects.filter(actors__id__in=actors_ids)
```

Comment:
You can simplify it into two conditions:
1. if genres_ids is not None
2. if actors_ids is not None
you can do:
movies = movies.filter....